{"name": "Проверка авторизации пользователя с неправильно указанным логином", "status": "passed", "description": "Если авторизоваться под несуществующим пользователем, запрос возвращает ошибку", "start": 1723215309903, "stop": 1723215310059, "uuid": "10c3e0a8-c25e-4bf0-a1b0-7ac27e897a66", "historyId": "53c588c18029e16555b57d963bc72e2f", "testCaseId": "53c588c18029e16555b57d963bc72e2f", "fullName": "tests.test_user.TestLoginUser#test_login_user_wrong_email_error_message", "labels": [{"name": "epic", "value": "test_login_user"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_user"}, {"name": "subSuite", "value": "TestLoginUser"}, {"name": "host", "value": "DESKTOP-619AU3M"}, {"name": "thread", "value": "12804-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_user"}]}